src/SFA.DAS.EAS.Web/FileDownloads/Payments/CsvTransactionFormatter.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using SFA.DAS.EAS.Domain.Models.Transaction;
+using SFA.DAS.EAS.Web.ViewModels;
+
+namespace SFA.DAS.EAS.Web.FileDownloads.Payments
+{
+    public class CsvTransactionFormatter : ITransactionFormatter
+    {
+        private static readonly string Headers = CreateHeaders();
+
+        public string MimeType => "text/csv";
+
+        public string FileExtension => "csv";
+
+        public DownloadFormatType DownloadFormatType =>
+            DownloadFormatType.Csv;
+
+        public byte[] GetFileData(List<TransactionDownloadLine> transactions)
+        {
+            var builder = new StringBuilder();
+
+            builder.AppendLine(Headers);
+
+            WriteRowsCsv(transactions, builder);
+			
+			var csvContent = builder.ToString();
+
+            return Encoding.UTF8.GetBytes(csvContent);
+        }
+
+        private static string CreateHeaders()
+        {
+            var headerBuilder = new StringBuilder();
+            headerBuilder.Append("Transaction date,");
+            headerBuilder.Append("Transaction type,");
+            headerBuilder.Append("PAYE scheme,");
+            headerBuilder.Append("Payroll month,");
+            headerBuilder.Append("Levy declared,");
+            headerBuilder.Append("English %,");
+            headerBuilder.Append("10% top up,");
+            headerBuilder.Append("Training provider,");
+            headerBuilder.Append("Cohort reference,");
+            headerBuilder.Append("Unique learner number,");
+            headerBuilder.Append("Apprentice,");
+            headerBuilder.Append("Apprenticeship training course,");
+            headerBuilder.Append("Paid from levy,");
+            headerBuilder.Append("Your contribution,");
+            headerBuilder.Append("Government contribution,");
+            headerBuilder.Append("Total,");
+
+            return headerBuilder.ToString();
+        }
+
+        private static void WriteRowsCsv(IEnumerable<TransactionDownloadLine> transactions, StringBuilder builder)
+        {
+            foreach (var transaction in transactions)
+            {
+                builder.Append($"{transaction.DateCreated:G},");
+                builder.Append($"{transaction.TransactionType},");
+                builder.Append($"{transaction.EmpRef},");
+                builder.Append($"{transaction.PeriodEnd},");
+                builder.Append($"{transaction.LevyDeclared.ToString(NumberFormatInfo.InvariantInfo)},");
```

Comment:
I think the .ToString() is redundant - the string interpolation is going to call that anyway.